frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SharingPaneButton/SharingPaneButton.tsx

```diff
@@ -20,32 +24,55 @@ export const SharingPaneButton = ({
   title,
   onClick,
   badge,
-  externalLink = false,
   "data-testid": dataTestId,
-}: SharingOptionProps) => (
-  <Paper
-    className={S.Container}
-    p={24}
-    pt={52}
-    withBorder
-    data-testid={dataTestId}
-    onClick={onClick}
-    mih="100%"
-    pos="relative"
-    w={"22rem"}
-  >
-    <Stack>
-      {externalLink && (
-        <Box pos="absolute" top={12} right={12}>
-          <Icon name="share" color="var(--external-link-icon-color)" />
+  isDisabled,
+  disabledLink,
+}: SharingOptionProps) => {
+  return (
+    <Paper
```

Comment:
there are some markup + css changes here for the disabled banner.
![Screenshot 2024-11-28 at 9 16 44 PM](https://github.com/user-attachments/assets/21bd0bd3-1d3b-4ea3-b44a-16491fe968c2)